Project/core/src/main/java/View/screen/MainMenuScreen.java

```diff
@@ -33,8 +36,28 @@ public void changed(ChangeEvent event, Actor actor) {
             }
         });
 
+        quitButton.addListener(new ChangeListener() {
+            @Override
+            public void changed(ChangeEvent event, Actor actor) {
+                Gdx.app.exit();
+            }
+        });
+
+        float width = stage.getCamera().viewportWidth;
+        float height = stage.getCamera().viewportHeight;
+
+        startButton.setSize(width * 0.1f, height * 0.05f);
```

Comment:
When I run the game the start button text doesn't really fit into the button. Maybe we could avoid setting the size and use their default size (which is based on the text size)? Or if we want them to have identical size we could use the start buttons default size as the size of the quit button?